lib/ui/questions/survey_questions_screen.dart

```diff
@@ -44,6 +45,10 @@ class _SurveyQuestionsScreenScreenState
 
   @override
   Widget build(BuildContext context) {
+    ref.listen(surveySubmittedStream, (previous, next) {
+      context.goNamed(RoutePath.submission.routePath);
```

Comment:
```suggestion
      context.goNamed(RoutePath.submission.routeName);
```
This should be `routeName`

Reply:
🙈